build.gradle

```diff
@@ -29,6 +29,9 @@ plugins {
 
     // https://docs.gradle.org/current/userguide/publishing_maven.html
     id "maven-publish"
+
+    // https://docs.gradle.org/current/userguide/idea_plugin.html
+    id "idea"
```

Comment:
Why this? Importing a Gradle project to IntelliJ should work fine?

Reply:
This is for integration tests, which will appear soon

Reply:
Removed for now, will be added back with those tests.